### ISSUE TITLE
Sign up page cleanup

### ISSUE THREAD

**Issue by jlian** [jlian/wingdex#382]
For the sign up view at some point we had very nice liquid glass buttons for google and github. I think they are now this always boring black. I think it might have to do with accessibility audit. Maybe loosen the accessibility audit and bring back the glass.

Also the sign up page currently hosts a bunch of errors in small red text that shift the layout. Should probably be moved to toasts. Also we should review the text copy.
